README.md

```diff
@@ -3,14 +3,20 @@
 The Uh!live Python SDK provides convenient access to the Uh!live API from
 applications written in the Python language.
 
-Read the [documentation for the Conversation API](https://docs.allo-media.net/live-api/) and [for the Recognition API (vocal bot toolkit)](https://docs.allo-media.net/stream-api-bots/).
+Read the [full documentation](https://python-uhlive-sdk.netlify.app/).
```

Comment:
Why? I can't access this website

Reply:
you're just reviewing it as a netlify preview before deployment :wink: but I already put the link in anticipation.

Reply:
We should create a DNS entry for this documentation. Something like https://docs.uh.live/sdk/python.
@shostelet I let you think about a proper structure for the URL

Reply:
Of course, I was not yet fully awake :sweat_smile: 
https://docs.uh.live/sdk/python suits me

Reply:
It must be a subdomain to work with Netlify, not a path @merwan

Reply:
We can use Netlify rewrite rules to setup such URL: https://docs.netlify.com/routing/redirects/rewrites-proxies/#proxy-to-another-netlify-site
I will try to setup https://docs.allo-media.net/sdk/python like this